sqlitedict.py

```diff
@@ -168,10 +168,14 @@ def __init__(self, filename=None, tablename='unnamed', flag='c',
         self.decode = decode
 
         logger.info("opening Sqlite table %r in %r" % (tablename, filename))
-        MAKE_TABLE = 'CREATE TABLE IF NOT EXISTS "%s" (key TEXT PRIMARY KEY, value BLOB)' % self.tablename
         self.conn = self._new_conn()
-        self.conn.execute(MAKE_TABLE)
-        self.conn.commit()
+        if self.flag == 'r':
```

Comment:
I forgot what flags we support, but maybe `if 'r' in self.flag` more future-proof?

Reply:
Sure, but there are some other code paths which uses == here so I suggest we keep this and refactor later?